navigation/create_control/launch/create_control.launch

```diff
@@ -0,0 +1,18 @@
+<launch>
+
+  <!-- Load controller configuration from YAML file to parameter server -->
+  <arg name="real_robot" value="$(optenv REAL_ROBOT False)"/>
+  <arg name="robot_id"  default="$(optenv ID 1)"      doc="Unique identifier of the robot [1-Inf.)"/>
+  <arg name="ns"        value="create$(arg robot_id)" doc="Namespace of the robot. By default: create1"/>
+
+  <node name="robot_hw_interface" pkg="create_control" type="create_control_node" if="$(arg real_robot)"
+    ns="$(arg ns)">
+    <rosparam command="load" file="$(find ca_driver)/config/default.yaml"/>
+  </node>
+  <rosparam file="$(find create_control)/config/create_2_control.yaml" command="load"/>
```

Comment:
Should this be moved inside the above node? If you do that, you will have to remove the `create1:` tag inside `create_2_control.yaml`.

Reply:
Doesn't work if so